docs/API.md

```diff
@@ -8,9 +8,9 @@ SEBAK, the next BOScoin network with ISAAC consensus protocol.
 <!-- partial(API_v1/models.md) -->
 <!-- partial(API_v1/transactions.md) -->
 
+
```

Comment:
Unneeded

Reply:
??? @Geod24  I think this comment for generating docs. isn't it??